topo_processor/metadata/loaders/metadata_loader_checksum.py

```diff
@@ -0,0 +1,16 @@
+from topo_processor.metadata.item import Item
+from topo_processor.util.checksum import multihash_as_hex
+from topo_processor.util.tiff import is_tiff
+
+from .metadata_loader import MetadataLoader
+
+
+class MetadataLoaderChecksum(MetadataLoader):
+    name = "loader.checksum"
+
+    def is_applicable(self, item: Item) -> bool:
+        return is_tiff(item.path)
+
+    def add_metadata(self, item: Item) -> None:
+        item.stac_item.stac_extensions.append("checksum")
```

Comment:
you need to check if it already exists.

Reply:
but isn't this the only place we are appending checksum?